app/models/bot/smooch.rb

```diff
@@ -712,6 +713,15 @@ def self.create_project_media(message, type, extra)
     pm
   end
 
+  def self.detect_media_type(message)
```

Comment:
I prefer the style of coding where we avoid side-effects on the input parameter, by explicitly returning the value - in this case the MIME type. E.g. `message['mediaType'] = self.detect_media_type(message)`

Reply:
Done